scout/server/blueprints/cases/views.py

```diff
@@ -49,6 +61,77 @@
 )
 
 
+def keymetrics_rows(samples_ids, genes=None):
+    """Generate key metrics rows."""
+    query = (
+        api.query(
+            TranscriptStat,
+            func.avg(TranscriptStat.mean_coverage).label("mean_coverage"),
+            func.avg(TranscriptStat.completeness_10).label("completeness_10"),
+            func.avg(TranscriptStat.completeness_15).label("completeness_15"),
+            func.avg(TranscriptStat.completeness_20).label("completeness_20"),
+            func.avg(TranscriptStat.completeness_50).label("completeness_50"),
+            func.avg(TranscriptStat.completeness_100).label("completeness_100"),
+        )
+        .with_hint(TranscriptStat, "USE INDEX (_sample_transcript_uc)")
```

Comment:
This is the fix that will be introduced in chanjo-report (see PR https://github.com/Clinical-Genomics/chanjo-report/pull/53)

Reply:
Well done! I think you can feel rather satisfied with that line of testing and problem solving! Simple fix in the end, but very much not obvious at first.

Reply:
> Well done! I think you can feel rather satisfied with that line of testing and problem solving! Simple fix in the end, but very much not obvious at first.

No, I feel like I can become an SQL consultant from now on! 😆